plexpy/http_handler.py

```diff
@@ -88,19 +88,25 @@ def make_request(self,
                 return None
 
             if request_status == 200:
-                if output_format == 'dict':
-                    output = helpers.convert_xml_to_dict(request_content)
-                elif output_format == 'json':
-                    output = helpers.convert_xml_to_json(request_content)
-                elif output_format == 'xml':
-                    output = helpers.parse_xml(request_content)
-                else:
-                    output = request_content
-
-                if return_type:
-                    return output, content_type
-
-                return output
+                try:
+                    if output_format == 'dict':
+                        output = helpers.convert_xml_to_dict(request_content)
+                    elif output_format == 'json':
+                        output = helpers.convert_xml_to_json(request_content)
+                    elif output_format == 'xml':
+                        output = helpers.parse_xml(request_content)
+                    else:
+                        output = request_content
+    
+                    if return_type:
+                        return output, content_type
+    
+                    return output
+                    
+                except Exception as e:
+                    logger.warn(u"Failed format response from uri %s to %s error %s. The original response was %s" % (uri, output_format, e, request_content))
```

Comment:
Not so sure we should log `request_content` here. That may be a very large amount of text.